examples/plot_mercator_argo.py

```diff
@@ -0,0 +1,230 @@
+import xoa.cf as xcf
+import xoa.coords as xcoords
+import xoa.geo as xgeo
+import xoa.plot as xplot
+
+mpl.rc("axes", grid=True)
+xr.set_options(display_style="text")
+
+# %%
+# Register the :ref:`xoa <accessors>` accessors :
+xoa.register_accessors()
+
+# %%
+# Register the Mercator and ARGO naming specifications
+xcf.register_cf_specs(xoa.get_data_sample("mercator.cfg"))
+xcf.register_cf_specs(xoa.get_data_sample("argo.cfg"))
+
+# %%
+# Here is what these CF specifications contain
+with open(xoa.get_data_sample("mercator.cfg")) as f:
+    print(f.read())
+with open(xoa.get_data_sample("argo.cfg")) as f:
+    print(f.read())
+
+# %%
+# Read and decode the datasets
+# ----------------------------
+
+# %%
+# ARGO profiles
+# ~~~~~~~~~~~~~
+#
+# The ARGO data were downloaded with the `argopy <https://argopy.readthedocs.io/en/latest/>`_
+# package with a piece of code close to the following::
+#
+#     from argopy import DataFetcher
+#     adf = DataFetcher().float(7900573)
+#     ds = adf.to_xarray().argo.point2profile().isel(N_PROF=slice(-10, None))
+#     ds.to_netcdf("argo-7900573.nc")
+ds_argo = xoa.open_data_sample("argo-7900573.nc")
+
+# %%
+# We decode the names to make them more generic thanks to the CF specifications
+# and select only the variables of interest.
+ds_argo = ds_argo.xoa.decode()[["temp", "sal", "pres"]]
+
+# %%
+# We compute depths with the :mod:`gsw` package and assign them
+# as coordinates of the ARGO dataset.
+alat = ds_argo.lat.broadcast_like(ds_argo.pres)
+adepth = gsw.z_from_p(ds_argo.pres, alat)
+adepth = adepth.where(adepth.notnull(), adepth.min())
+adepth.attrs.update({"long_name": "Depth", "units": "m"})
+ds_argo = ds_argo.assign_coords(depth=adepth)
+ds_argo = ds_argo.isel(level=slice(None, None, -1))
+ds_argo = ds_argo.set_index(N_PROF="time").rename(N_PROF="time")
+
+# %%
+# Quick plot of the salinity that highligths the mediterranean water.
+fig, axs = plt.subplots(ncols=2, figsize=(14, 6))
+ds_argo.sal.plot.contourf("time", "depth", cmap="cmo.haline", levels=20, ax=axs[0])
+ds_argo.sal.plot.contour("time", "depth", levels=[35.65], linewidths=1, colors="k", ax=axs[0])
+xplot.plot_ts(
+    ds_argo.temp,
+    ds_argo.sal,
+    potential=False,
+    scatter_c=ds_argo.depth,
+    scatter_cmap="cmo.deep",
+    axes=axs[1],
+)
+axs[1].axvline(x=35.65, ls="--", color="k")
+
+# %%
+# Here we select the last ARGO profile as a reference profile
+# since it better samples de mediterranean water vein.
+ds_argo_prof = ds_argo.isel(time=-1).squeeze(drop=True)
+
+# %%
+# Mercator
+# ~~~~~~~~
+#
+# The Mercator data come from the IBI configuration and were download
+# from the CMEMS site and undersampled by a factor 3 in longitude and latitude.
+ds_merc = xoa.open_data_sample("ibi-argo-7900573.nc").xoa.decode()
+ds_merc = ds_merc.isel(depth=slice(None, None, -1))
+ds_merc = ds_merc.assign_coords(depth=-ds_merc.depth)
+print(ds_merc)
+
+# %%
+# This block of data can be viewed as a ensemble of profiles that emulates
```

Comment:
a set of profiles ?

Reply:
"ensemble" refers to the ensemble data assimilation in which the "set" emulates the uncertainties.